src/olympia/addons/models.py

```diff
@@ -153,13 +153,9 @@ def id_or_slug(self, val):
             return self.filter(slug=val)
         return self.filter(id=val)
 
-    def enabled(self):
-        """Get add-ons that haven't been disabled by their developer(s)."""
-        return self.filter(disabled_by_user=False)
-
     def public(self):
-        """Get public add-ons only"""
-        return self.filter(self.valid_q([amo.STATUS_APPROVED]))
+        """Get reviewed add-ons only"""
+        return self.filter(self.valid_q(amo.REVIEWED_STATUSES))
```

Comment:
The issue says this is just removing unused code, but it looks like the logic of this has changed as well. Can you explain that?

Reply:
in the constants, `REVIEWED_STATUSES = (STATUS_APPROVED,)` so it's the same.